api/AudioParam.json

```diff
@@ -393,7 +393,7 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "24"
```

Comment:
Before Chrome 24, it was called `setTargetValueAtTime`.

- Implemented in https://github.com/WebKit/WebKit/commit/ab9cb22694d752aac76b4a08df906df524d8896c (WebKit [535.1](https://github.com/WebKit/WebKit/blob/ab9cb22694d752aac76b4a08df906df524d8896c/Source/WebCore/Configurations/Version.xcconfig#L24-L26)).
- Renamed in https://github.com/WebKit/WebKit/commit/525fa1cd4c3de244719ced162e1c89829d1717a2 (WebKit [537.16](https://github.com/WebKit/WebKit/blob/525fa1cd4c3de244719ced162e1c89829d1717a2/Source/WebCore/Configurations/Version.xcconfig#L24-L26))